homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -187,13 +192,18 @@ def _update_current_obs(self, entity_observation, should_trigger):
                 entity_id = entity_observation.get(
                     CONF_VALUE_TEMPLATE
                 ).extract_entities()
+
             self.current_obs[obs_id] = {
                 "prob_true": prob_true,
                 "prob_false": prob_false,
+            }
+            self.current_ent[obs_id] = {
```

Comment:
Why would we track it and untrack it? We should just extract `entity_id` from the (numeric) state schemas and set that as an attribute

Reply:
Done